### ISSUE TITLE
Will you accept a PR that allows setting minimum width/height for resizable windows?

### ISSUE THREAD

**Issue by medvednikov** [floooh/sokol#1012]
Quite a useful feature, would be nice to have it upstream.

**Comment by floooh** [comment]
Yes if the code isn't too gnarly. Probably best to look at glfwSetWindowSizeLimits() (although tbh, the GLFW code isn't too straightforward).

I'm not sure if it's better to have fixed limits provided in `sapp_desc`, or a function `sapp_set_window_size_limits()`, or both.

**Comment by medvednikov** [comment]
I added `min_width` and `min_height` to `sapp_desc`.
